inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -877,82 +877,76 @@ void InsertCopyLayerPass::run() {
     }
 }
 
-void Concat4Dto2DPass::run() {
-    // Find 4D concat layers that will have to use ConcatAlignFilters and can be substituted by 2D concat
-    // for example if 4D concat have unaligned inputs then ConcatAlignFilters need to be used if sizes before
-    // axis are all ones then concat can be changed to 2D for example, lets say all unputs have same shape equal to:
-    // 1, 1, 5, 3 then for axis 0, 1, 2 the change will be made and inputs will be reshaped to 1, 15,
-    // but for shape 2, 1, 5, 3 only axis 0 is valid and inputs will reshape to 1, 30
+void FlattenTrivialConcatPass::run() {
+    // change all trivial concatenations (concatenation where output buffer is a buffer made by appending input buffers)
```

Comment:
The example was good as it shows what this pass does

Reply:
Good point, it was 4D specific so I removed it but, now I think about it there is not problem with example being 4D specific